doc/integrations/Single-Sign-On/Github.md

```diff
@@ -0,0 +1,18 @@
+## Configuring Github Enterprise Single-Sign-On
```

Comment:
Since the GitLab SSO UI is used for GitHub Enterprise SSO, it's potentially confusing. Propose adding text to explain that this feature is `unofficial`. In future, after SSO is refactored to be more modular, we would add the appropriate UI and update the documentation. 

```
## Configuring GitHub Enterprise Single-Sign-On (unofficial) 

Note: Because the authentication interface of GitHub Enterprise is similar to that of GitLab, the GitLab SSO feature can be used to unofficially also support GitHub Enterprise SSO. 

Follow these steps to configure Mattermost to use Github Enterprise as a single-sign-on (SSO) service for team creation, account creation and sign-in using the GitLab SSO interface. 

```